lib/key-agreement-key.js

```diff
@@ -58,9 +49,7 @@ exports.generateKeyPair = async ({keyId, type}) => {
       }, e);
   }
 
-  const publicKeyNode = {...key};
-  delete publicKeyNode.privateKeyBase58;
-  return publicKeyNode;
+  return {...keyPair.export({publicKey: true})};
```

Comment:
Looks like this could be simplified:
```suggestion
  return keyPair.export({publicKey: true});
```

Reply:
Changed, thanks.